pkg/kubelet/cm/topologymanager/fake_topology_manager.go

```diff
@@ -29,30 +31,30 @@ type fakeManager struct {
 }
 
 // NewFakeManager returns an instance of FakeManager
-func NewFakeManager() Manager {
-	klog.InfoS("NewFakeManager")
+func NewFakeManager(ctx context.Context) Manager {
```

Comment:
This change seems to cause most of the changes in other components. Would it make sense to postpone it/use `context.TODO` with a comment?

Reply:
I unfortunately don't have more bandwidth to devote to this CL. The changes are already done and any resulting TODOs spawning from this are easy enough to find.

Reply:
Yes, but without so many changes in other components it would be easier to understand, review and merge.

Reply:
I'll take it over if you don't mind. Let me know if you're going to continue with it.